src/scipp/io/csv.py

```diff
@@ -0,0 +1,144 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2023 Scipp contributors (https://github.com/scipp)
+
+"""Load CSV files.
+
+Note
+----
+CSV support requires `pandas <https://pandas.pydata.org/>`_ to be installed.
+You need to do this manually as it is not declared as a dependency of Scipp.
+You can use either ``pip install pandas`` or ``conda install -c conda-forge pandas``.
+
+
+CSV ('comma separated values') files store a simple table of data as a string.
+There are many different forms of this format.
+So check whether the loaded data is what you expect for your files.
+See ``scipp.io.csv.load_csv`` for examples.
```

Comment:
`load_csv` is imported into `io`, do we need to refer to the `csv` submodule here?

Reply:
Not, not in this form here. But unfortunately, it is required for actual links. I changed this into a link. I don't think the full path matters here as this docstring is already in the csv package.

But it would be great to find a way to use shorter links, e.g., also `scipp.compat.pandas_compat.from_pandas` further down.